dashboard/app/api.go

```diff
@@ -771,8 +771,10 @@ func reportCrash(c context.Context, build *Build, req *dashapi.Crash) (*Bug, err
 
 	newSubsystems := []*subsystem.Subsystem{}
 	// Recalculate subsystems on the first saved crash and on the first saved repro.
```

Comment:
This comment looks outdated now. Remove?

Reply:
Updated the comment.